packages/plugin-ext/src/main/browser/view/tree-views-main.ts

```diff
@@ -108,6 +108,28 @@ export class TreeViewsMainImpl implements TreeViewsMain, Disposable {
         }
     }
 
+    async $setMessage(treeViewId: string, message: string): Promise<void> {
+        const viewPanel = await this.viewRegistry.getView(treeViewId);
+        const widget = viewPanel && viewPanel.widgets[0];
+
+        if (widget instanceof TreeViewWidget) {
+            widget.message = message;
+        } else {
+            console.error(`Widget not found for Tree View ID: ${treeViewId}`);
+        }
+    }
+
+    async $setTitle(treeViewId: string, title: string): Promise<void> {
+        const viewPanel = await this.viewRegistry.getView(treeViewId);
+        const widget = viewPanel && viewPanel.widgets[0];
+
+        if (widget instanceof TreeViewWidget) {
+            widget.message = title;
+        } else {
```

Comment:
the same as for message

Reply:
Agreed.

Actually there is an error here. Sorry, the title is to be set differently. Fixed